app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1679,5 +1679,526 @@
                 </browser_capabilities>
             </groups>
         </web>
+        
+        <advanced_grid translate="label" module="core">
+            <label>Admin Grids</label>
+            <tab>advanced</tab>
+            <sort_order>21</sort_order>
+            <show_in_default>1</show_in_default>
+            <show_in_website>0</show_in_website>
+            <show_in_store>0</show_in_store>
+            <groups>
+                <productGrid>
+                    <label>Catalog > Manage Products</label>
+                    <sort_order>1</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>0</show_in_website>
+                    <show_in_store>0</show_in_store>
+                    <fields>
+                        <enabled>
+                            <label>Enable</label>
```

Comment:
All **Enable** words must be replaced with **Enabled** to follow the Backend rule related to activate a feature with two options Yes/No.